site/docs/numbers/downloadingReports.mdx

```diff
@@ -133,7 +133,7 @@ Content-Type: application/xml; charset=utf-8
 ```
 
 ### Download the Report {#download-account-report}
-Once the report status query returns `Ready`, we are ready to download the requested report. This can be done by making a <code class="get">GET</code> request to the URI received in the location header of the response to our original <code class="post">POST</code> request, and appending it with `/file` . You can receive one of two successful responses: `200 OK` or `307 Temporary Redirect`.  
+Once the report status query returns `Ready`, we are ready to download the requested report. This can be done by making a <code class="get">GET</code> request to the URI received in the location header of the response to our original <code class="post">POST</code> request, and appending it with `/file`. Ensure that your client is configured to omit the `Authorization` header when following redirects returned by this endpoint, as the redirect URL includes auth in the query parameters. Requests to the redirect url that include multiple auth mechanisms will be rejected with `400 Bad Request`. You can receive one of two successful responses: `200 OK` or `307 Temporary Redirect`.  
```

Comment:
maybe it should specifically say that the redirect may be to a different origin, and the `Authorization` header should be omitted cross-origin?